src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -93,7 +95,7 @@ public function handle(GetResponseEvent $event)
         $request->query->remove($this->usernameParameter);
         $request->server->set('QUERY_STRING', http_build_query($request->query->all()));
 
-        $response = new RedirectResponse($request->getUri(), 302);
+        $response = new RedirectResponse($request->getUri(), RedirectResponse::HTTP_FOUND);
```

Comment:
-1 for this. We already decided in the past that we would not use the constants inside the Symfony code itself. We found using the integer status code more readable (`302` is what is important, not `HTTP_FOUND`)